oak_restricted_kernel/src/syscall/mod.rs

```diff
@@ -95,6 +95,16 @@ extern "sysv64" fn syscall_handler(
     arg5: usize,
     arg6: usize,
 ) -> isize {
+    log::info!(
```

Comment:
I am not convinced we should always be logging this.

Reply:
Oh, crap. This was a debug thing I didn't mean to include in this PR.